### DISCUSSION TITLE
Local Private Files for FileUpload?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#9386] [General]
How can i use the FileUpload with private files localy?

When i upload the Image, the file cant not be found.
"https://filamentplayground.ddev.site/storage/form-attachments/Co3KaRspw1EA7LCO5agDYir7mX1RsU-metabG9naW5fYmcuanBn-.jpg"

```
'default_filesystem_disk' => env('FILAMENT_FILESYSTEM_DISK', 'private'),
```

```
FileUpload::make('attachment')
                    ->image()
                    ->visibility('private')
                    ->directory('form-attachments')
```
    'private' => [
        'driver' => 'local',
        'root' => storage_path('app/private'),
        'url' => env('APP_URL').'/storage',
        'visibility' => 'private',
        'throw' => true,
    ]
```

**Comment by leandrocfe** [comment]
https://discord.com/channels/883083792112300104/1145723690839973908/1145725698229665886
